DigitalLearningSolutions.Data/DataServices/GroupsDataService.cs

```diff
@@ -74,15 +76,43 @@ public IEnumerable<GroupDelegate> GetGroupDelegates(int groupId)
             );
         }
 
-        public string GetGroupNameForGroupId(int groupId)
+        public IEnumerable<GroupCourse> GetGroupCourses(int groupId, int centreId)
+        {
+            return connection.Query<GroupCourse>(
+                @"SELECT
+                        GroupCustomisationID,
+                        GroupID,
+                        gc.CustomisationID,
+                        ap.ApplicationName,
+                        CustomisationName,
+                        Mandatory AS IsMandatory,
+                        IsAssessed,
+                        AddedDate AS AddedToGroup,
+                        au.Forename AS SupervisorFirstName,
+                        au.Surname AS SupervisorLastName,
+                        gc.CompleteWithinMonths,
+                        ValidityMonths
+                    FROM GroupCustomisations AS gc
+                    JOIN Customisations AS c ON c.CustomisationID = gc.CustomisationID
+                    INNER JOIN dbo.CentreApplications AS ca ON ca.ApplicationID = c.ApplicationID
+                    INNER JOIN dbo.Applications AS ap ON ap.ApplicationID = ca.ApplicationID
+                    LEFT JOIN AdminUsers AS au ON au.AdminID = gc.SupervisorAdminID
+                    WHERE gc.GroupID = @groupId
+                        AND ca.CentreId = @centreId
+                        AND InactivatedDate IS NULL",
+                new { groupId, centreId }
+            );
+        }
+
+        public string? GetGroupNameForGroupIdAndCentreId(int groupId, int centreId)

```

Comment:
We could probably drop the "ForGroupIdAndCentreId" as it's clear from params of the method - so simply GetGroupName is fine and easier to read. 👍